internal/pkg/cosign/tsa/signer.go

```diff
@@ -58,7 +60,7 @@ func GetTimestampedSignature(sigBytes []byte, tsaClient *tsaclient.TimestampAuth
 		return nil, err
 	}
 
-	fmt.Fprintln(os.Stderr, "Timestamp fetched with time:", ts.Time)
+	fmt.Fprintln(os.Stderr, "Timestamp fetched with time: ", ts.Time)
```

Comment:
This is how it looks today: `Timestamp fetched with time: 2022-12-01 17:44:21 +0100 CET` This will add an extra whitespace here.

Reply:
If this is meant to be debug output, I think it's more readable with whitespace, but I'm not super picky if y'all think otherwise.

Reply:
It is fine, I wanted to be sure we are aware that there is a whitespace between the time and the text.

Reply:
+1 to adding the whitespace